File: csharp/ghoul.cs
using System;

class DeadInside {  
    static void Main(string[] args)
    {
        for (int i = 1000; i > 2; i-=7)
        {
            if(i-7==-1) {
                Console.WriteLine("6 - 7 = 0");
                System.Threading.Thread.Sleep(1000);
                Console.WriteLine("I'm Ghoul");
                break;
            }    
            Console.WriteLine(i + " - 7 = " + (i - 7));
            System.Threading.Thread.Sleep(25);            
        }
    }       
}
    

File: javas/Ghoul.java
package javas;

public class Ghoul {
    
    public static void main(String[] args) throws InterruptedException {
        int ghoul = 1000;
        int minus = 7;

        while(ghoul>1){
            ghoul-=minus;
            if(ghoul-7==-1) {
                System.out.println("6 - 7 = 0\nI'm Ghoul");
                return;
            }
            System.out.println(Integer.toString(ghoul) + " - 7 = " + Integer.toString(ghoul-7));
            Thread.sleep(25);
        }


    }

}
